packages/backend/src/services/UserAttributesService/UserAttributeUtil.mocks.ts

```diff
@@ -149,6 +149,18 @@ export const EXPLORE_WITH_NO_REQUIRED_ATTRIBUTES: Explore = {
     },
 };
 
+if (!EXPLORE_WITH_NO_REQUIRED_ATTRIBUTES.tables.orders) {
+    throw new Error('Orders are undefined');
+}
+
+if (!EXPLORE_WITH_NO_REQUIRED_ATTRIBUTES.tables.payments) {
+    throw new Error('Orders are undefined');
+}
+
+if (!EXPLORE_WITH_NO_REQUIRED_ATTRIBUTES.tables.payments.dimensions.name) {
+    throw new Error('Name is undefined');
+}
```

Comment:
Having these checks on the file root is confusing. Since these are mocks, we can add `!` when using those properties.